packages/react-reconciler/src/forks/ReactFiberHostConfig.custom.js

```diff
@@ -54,6 +54,7 @@ export const shouldSetTextContent = $$$hostConfig.shouldSetTextContent;
 export const createTextInstance = $$$hostConfig.createTextInstance;
 export const scheduleTimeout = $$$hostConfig.scheduleTimeout;
 export const cancelTimeout = $$$hostConfig.cancelTimeout;
+export const queueMicrotask = $$$hostConfig.queueMicrotask;
```

Comment:
Can you please briefly document it in the `react-reconciler` README?

Reply:
Done 👍